src/compute/src/render/reduce.rs

```diff
@@ -2336,6 +2336,25 @@ mod monoids {
         }
     }
 
+    impl Clone for ReductionMonoid {
+        fn clone(&self) -> Self {
+            use ReductionMonoid::*;
+            match self {
+                Min(row) => Min(row.clone()),
+                Max(row) => Max(row.clone()),
+            }
+        }
+
+        fn clone_from(&mut self, source: &Self) {
+            use ReductionMonoid::*;
+            match (self, source) {
```

Comment:
This needs to make sure that the receiver inherits the correct enum variant from other, too.